### ISSUE TITLE
Autotator'd as borg without getting law zero

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#20993]
(WEB REPORT BY: kitfox88 REMOTE: 206.221.180.138:7777)
# Revision
9b1f59d0b44f6ca7e93f7753307deec0674e47f8

# Description
see title
# Steps to Reproduce
start as borg and get the cool syndicate voice clip
# What you Expected
murderbone gone wild wooo
# What Actually Happened
forced to be a proper crewmember by the system, maaaaannnn

**Comment by Pathid** [comment]
https://github.com/vgstation-coders/vgstation13/blob/eacfee5c088748f840e15c85ba343823b068a8d5/code/game/jobs/job_controller.dm#L442

**Comment by Kitfox88** [comment]
There was no AI that round, if it would matter

**Comment by Pathid** [comment]
It wouldn't matter either way. There's a delay before you get turned into a borg by the job controller _just in case_ there's an antag AI and it makes you miss your chance to get your law zero. I think.

**Comment by Intigracy** [comment]
Yeah it helped fix the "borgs not getting the tater/malf AI's law 0" bug before datums came around.

now it's back in full force
